src/test/scala/mesosphere/marathon/state/VolumeTest.scala

```diff
@@ -148,17 +148,17 @@ class VolumeTest extends UnitTest {
     }
 
     "validating that DiskSource asMesos converts to an Option Mesos Protobuffer" in {
-      DiskSource(DiskType.Root, None, None, Map.empty, None).asMesos shouldBe None
-      val Some(pathDisk) = DiskSource(DiskType.Path, Some("/path/to/folder"), None, Map.empty, None).asMesos
+      DiskSource(DiskType.Root, None, None, None, None).asMesos shouldBe None
+      val Some(pathDisk) = DiskSource(DiskType.Path, Some("/path/to/folder"), None, None, None).asMesos
```

Comment:
for these tests that use `None` for metadata .. shouldn't they verify that source.hasMetadata returns false?

Reply:
Done.